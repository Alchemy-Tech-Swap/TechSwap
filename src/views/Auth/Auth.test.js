import Auth from './Auth';
import { render, screen } from '@testing-library/react';
import { MemoryRouter, Route } from 'react-router-dom';

test('Auth.js does some stuff', async () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/sign-in']}>
      <Route path="/sign-in">
        <Auth />
      </Route>
    </MemoryRouter>
  );

  await screen.findAllByText('Sign In');

  expect(container).toMatchSnapshot();
});
