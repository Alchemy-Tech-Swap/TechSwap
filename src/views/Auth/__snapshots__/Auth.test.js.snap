// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth.js does some stuff 1`] = `
<div>
  <div
    class="Link"
  >
    <br />
    <br />
    <div
      classnames="Link"
    >
      <h1
        class="active signInHeader"
      >
        Sign In
      </h1>
      <h1
        class="signUpHeader"
      >
        Sign Up
      </h1>
    </div>
    <p>
       
      Sign In
       
    </p>
    <form
      class="form"
    >
      <h3 />
      <div
        class="form-controls"
      >
        <label>
          Email:
        </label>
        <input
          placeholder="Enter Valid Email Address"
          type="email"
          value=""
        />
      </div>
      <div
        class="form-controls"
      >
        <label>
          Password:
        </label>
        <input
          placeholder="Minimum 6 digits req."
          type="password"
          value=""
        />
      </div>
      <input
        class="submit"
        type="submit"
      />
    </form>
  </div>
</div>
`;
